src/js/components/SearchBar/Error.js

```diff
@@ -0,0 +1,41 @@
+/* @flow */
+
+import {useSelector} from "react-redux"
+import React from "react"
+import styled from "styled-components"
+
+import type {Styled} from "../../types/styled"
+import SearchBar from "../../state/SearchBar"
+import Warning from "../icons/warning-sm.svg"
+
+const Wrap: Styled<> = styled.div`
```

Comment:
The error message that appears if the syntax is invalid.

<img width="869" alt="image" src="https://user-images.githubusercontent.com/3460638/91352717-11a40680-e79f-11ea-8d31-604f8d8b4594.png">